src/app/core/models/button.enum.ts

```diff
@@ -1,18 +1,18 @@
 export enum buttonType {
-    fill,
```

Comment:
@MirandaMakharashvili @grgpk looks like we have style conflicts, make sure you have same settings for this